src/schema/objects/metadata.yaml

```diff
@@ -3780,6 +4007,27 @@ StimulusPresentation:
       $ref: objects.metadata.SoftwareVersion
     Code:
       $ref: objects.metadata.Code
+StimulusPresentation__eyetrack:
+  $ref: objects.metadata.StimulusPresentation
+  required:
+    - ScreenDistance
```

Comment:
per [comment above](https://github.com/bids-standard/bids-specification/pull/1128/files#r2411009444) - might be recordings "in the wild" without screen, thus I would move all those into `recommended`. 

Otherwise have to be explicitly `"n/a"` for such experiments?

Reply:
oh we arrived to the same conclusion, ah ah ...